pkg/blobfuse-proxy/README.md

```diff
@@ -1,49 +1,47 @@
-### Introduction
+# Use blobfuse-proxy
 
-Currently there is (not easy way)[https://github.com/kubernetes/kubernetes/issues/70013] to update CSI drivers that use FUSE. Because when a nodeserver restarts, the FUSE daemon also restarts. This results in breaking all this connections the FUSE daemon maintains. So one workaround is to run a fuse proxy
-on all the host machines and this proxy mounts volumes and maintains FUSE connections.
+By default, restart csi-blobfuse-node daemonsetwould make current blobfuse mounts unavailable. When fuse nodeserver restarts on the node, the fuse daemon also restarts, this results in breaking all connections FUSE daemon is maintaining. You could find more details here: [No easy way how to update CSI driver that uses fuse](https://github.com/kubernetes/kubernetes/issues/70013).
 
-Blobfuse proxy receives mount request in a GRPC call and then uses this data to mount and returns the output of the blobfuse command.
+This page shows how to run a blobfuse proxy on all agent nodes and this proxy mounts volumes and maintains FUSE connections. Blobfuse proxy receives mount request in a GRPC call and then uses this data to mount and returns the output of the blobfuse command.
 
+### Prerequisite
+ - make sure [blobfuse](https://github.com/Azure/azure-storage-fuse) is already installed on agent node
 
-### Usage
+### Install blobfuse-proxy on debian-based agent node
```

Comment:
@boddumanohar could you take a review at this blobfuse-proxy doc change? also `./hack/install-protoc.sh` actually does not work now.

Reply:
Thanks for pointing it out. I have raised a PR to fix the failures.